Tests/SwiftFormatPrettyPrintTests/IfConfigTests.swift

```diff
@@ -390,4 +390,68 @@ final class IfConfigTests: PrettyPrintTestCase {
 
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 45)
   }
+
+  func testPostfixPoundIfBetweenOtherModifiers() {
+    let input =
+      """
+      EmptyView()
+        .padding([.vertical])
+      #if os(iOS)
+        .iOSSpecificModifier()
+      #endif
+        .commonModifier()
+      """
+
+    let expected =
+      """
+      EmptyView()
+        .padding([.vertical])
+      #if os(iOS)
+        .iOSSpecificModifier()
+      #endif
+        .commonModifier()
```

Comment:
This is not handled properly by these changes because I didn't think to try adding this until I was creating this pull request. Will look into how to solve this but wanted to have it in this pull request so I didn't forget to mention it.

Reply:
Have you had any luck with this one? For consistency we'd definitely like if the `#if` aligns with the preceding `.` and then the contents are indented one more level.

Reply:
I have not yet, things at work got really busy soon after I created this. I'll take another look at it soon.

Reply:
Took me a bit longer than I expected to get back to this. I think [this latest commit](https://github.com/apple/swift-format/pull/402/commits/d7b390fafce18fa8e734c196a6a90fbde430f8a0) works as intended. Let me know if I misunderstood.

I don't love the implementation I landed on so I'l be looking to improve it if I can.

Reply:
I'm happier where this is now. Would appreciate any and all feedback :)

Reply:
@allevato Friendly ping for awareness, not urgency, here since it took me so long to get back to this. I think it is behaving how you described now.

Reply:
Sorry for the delay; I lost track of this one. These results look great, thanks for getting all these cases working so well!